articles/network-types-topologies/index.md

```diff
@@ -0,0 +1,91 @@
+
+![Example of WAN](/engineering-education/network-types-topologies/wan.jpg)
+
+ Configuring a computer network can also depend on the extent of the network, the distance between devices and the number of devices used. While LANs, WLANs, and WANs are most commonly used, there are some [other](https://www.belden.com/blog/smart-building/network-types) computer networks that can also be used based on our networking parameters.
+
+
+
+### Network Topologies
+Arrangement of components or parts in a specific way is called a **Topology.** In [network topologies](https://en.wikipedia.org/wiki/Network_topology), the components are arranged in a systematic order for *smooth data flow* in a communication network. Here, the components may be devices, workstations, etc which are also referred to as **nodes**. Many topologies are used based on their applications, structure and efficiency. While some are used by commercial establishments and industrial sectors, some are used by schools and for household purposes.
+
+#### Linear Topology
+This type of topology also goes by the name **Bus Topology**. The name itself states that, *the nodes or components of a system are connected in a linear manner*. All the nodes are connected to one bus(cable) in which the data runs linearly from the start point to the endpoint. Terminators are attached at both ends to define the length of the bus. All the nodes are connected to the bus within the two terminators. So there is a *bidirectional* flow of data.
+The arrangement of bus topology is very simple and direct. Since only one transmission medium is used, the data is easily accessible. Due to its simple architecture, the extension of bus topology becomes easier. Since a single cable is used, when two workstations send data simultaneously they may hit one another. This leads to an increase in network traffic. Also, adding new devices and components may slow down the data transfer rate.
+
+![Linear Topology](/engineering-education/network-types-topologies/bus.jpg)
+
+#### Ring Topology
+Ring topology is a special kind of bus topology. *A linear topology in which both ends are joined together forms a **Ring Topology***. Thus no terminators are used and so the data flow is *unidirectional*. Because of that, for the data to reach from the first node to the last  it has to traverse through all the nodes. Since the connection of nodes is continuous, data flows infinitely throughout the topology until one node removes it.
+Like linear topology, the structure and arrangement of nodes are very simple. Due to its compact architecture, the configuration of nodes becomes easier. Since the nodes are connected in a loop, unpredictable troubleshooting problems may appear. To repair or fix one crashed node, the whole system has to be taken down. It is difficult to shut down one node and keep the rest of the system functional. Like the linear topology, adding new devices and components may slow down the data transfer rate.

```

Comment:
Same as the comment above - I think you've already stated some advantages and disadvantages but making it clear to the reader for help with flow and readability. 
Also is this network type used much today?

Reply:
Sure- I'll do one thing. I'll make 2 separate parts(Advantages and Disadvantages) for each topology and list the points there.

Well, kind of. The data sent by the source computer passes through all the computers to reach the destination computer. This allows all the computers to "see" what the data is. This helps when data has to be shared publicly with more people. Like in our school, teachers used to share these lecture slides and PPTs from their computer(source: teacher) to all the computers simultaneously and they would start appearing one by one in our computer(destination: student) as they traversed throughout the system. Our teachers said that all the computers were connected in a ring manner. But nowadays, star topology is considered to be a leading structure/model to form a communication network. Since a central hub is used, the administrator controlling the hub decides what data to send to which computer, which is kind of risk-free and reliable.
So basically, ring topology is not suited for commercial and business purposes. But it's ok to use when the data security is not an issue or the transmission is between trusted computers. Also, ring is cheaper and requires less cabling than compared to star.